injective_functions/exchange/exchange.py

```diff
@@ -19,29 +19,28 @@ def __init__(self, chain_client) -> None:
         super().__init__(chain_client)
 
     async def get_subaccount_deposits(
-        self, subaccount_idx: int, denoms: str = None
+        self, subaccount_idx: int, denoms: List[str] = None
```

Comment:
Denoms can be optional yes? Let's mark it as such

Reply:
weird, i did initally set that to none